cmd/platform/export.go

```diff
@@ -108,28 +106,31 @@ func initGeneralExportFlags(cmd *cobra.Command) {
 }
 
 func initPingOneExportFlags(cmd *cobra.Command) {
-	cmd.Flags().AddFlag(options.PingOneAuthenticationWorkerEnvironmentIDOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationAuthCodeClientIDOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationAuthCodeEnvironmentIDOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationAuthCodeRedirectURIOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationAuthCodeScopesOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationClientCredentialsClientIDOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationClientCredentialsClientSecretOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationClientCredentialsEnvironmentIDOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationClientCredentialsScopesOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationDeviceCodeClientIDOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationDeviceCodeEnvironmentIDOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationDeviceCodeScopesOption.Flag)
+	cmd.Flags().AddFlag(options.PingOneAuthenticationTypeOption.Flag)
 	cmd.Flags().AddFlag(options.PingOneAuthenticationWorkerClientIDOption.Flag)
```

Comment:
What's the difference between these and the `PingOneAuthenticationClientCredentialsClientIDOption` and `PingOneAuthenticationClientCredentialsClientSecretOption` flags?